flottform/demo/src/routes/turn-server-control/+page.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+    import { base } from '$app/paths';
+	import Logo from '$lib/components/Logo.svelte';
+
+    let authKey = '';
+    let useTurnServer = false;
+    let message = '';
+  
+    const handleSubmit = async (e:Event) => {
+      e.preventDefault()
+
+      if (!authKey) {
+        message = 'Please provide an authentication key.';
+        return;
+      }
+  
+      try {
+        const response = await fetch('https://192.168.0.167:5177/flottform/turn-control', {
```

Comment:
Needs to use a variable (which in turn should be an environment variable). I guess this may be something like `window.location.origin` in the default case instead of a private IP